roles/ansible-runner/tasks/runner.yml

```diff
@@ -0,0 +1,49 @@
+- name: Download source
+  git:
+    repo: "{{ item.repo }}"
+    dest: "/opt/source/{{ item.name }}"
+    update: true
+
+- name: Install config file
+  template:
+    src: ansible-runner-config
+    dest: /etc/default/{{ item.name }}
+    mode: 0600
+    owner: "{{ item.user }}"
+
+- name: Ensure source ownership
+  file:
+    dest: "/opt/source/{{ item.name }}"
+    owner: "{{ item.user }}"
+    recurse: yes
+
+- name: Check for playbook deps
+  stat:
+    path: "/opt/source/{{ item.name }}/requirements.txt"
+  register: requirements_txt
+
+- name: Install playbook deps
+  pip:
+    virtualenv: "{{ ansible_runner_virtualenv }}"
+    requirements: "/opt/source/{{ item.name }}/requirements.txt"
+  when: requirements_txt.stat.exists
+
+- name: Ensure cron log path
+  file:
+    state: directory
+    path: "/var/www/html/cron-logs/{{ item.name }}"
+    mode: 0755
+    owner: "{{ item.user }}"
+    group: www-data
```

Comment:
Should this go after the `include: secret-file.yml`? Just to help keep the tasks organized.